ci/build_alpine_container/Makefile

```diff
@@ -0,0 +1,28 @@
+PHONY: all
+
+ENVOY_TAG?="latest"
+
+all: binary container test
+
+binary:
+	rm -rf ./bin
+	mkdir -p ./bin/stripped
+	mkdir -p ./bin/configs
+	docker run -t -i --name build-envoy-alpine-ct -v ${PWD}/../../:/source lyft/envoy-build:latest /bin/bash -c \
+		"set -x && cd /source && /source/ci/do_ci.sh normal && \
```

Comment:
May be its a conversation for another thread. But just in case, @htuch we are still retaining Dockerfile based build of Envoy images right? Bazel's support for building docker images is at odds with what Docker expects. The images turn out to be 200 years old :P.